content/patterns/combobox/examples/js/select-only.js

```diff
@@ -239,9 +240,10 @@ Select.prototype.getSearchString = function (char) {
   return this.searchString;
 };
 
-Select.prototype.onComboBlur = function () {
+Select.prototype.onComboBlur = function (event) {
   // do not do blur action if ignoreBlur flag has been set
-  if (this.ignoreBlur) {
+  // also do not blur if focus is moving to the listbox
+  if (this.ignoreBlur || this.listboxEl.contains(event.relatedTarget)) {
```

Comment:
you can actually remove all `ignoreBlur` logic now, since it was added to prevent the listbox from dismissing when clicking on an option. Now the `this.listboxEl.contains(event.relatedTarget)` will handle that, so `ignoreBlur` + the option mousedown event handler isn't necessary :)

Reply:
oh thanks. just updated